temporalio/worker/_interceptor.py

```diff
@@ -314,6 +326,14 @@ async def handle_query(self, input: HandleQueryInput) -> Any:
         """Called to handle a query."""
         return await self.next.handle_query(input)
 
+    async def handle_update_validator(self, input: HandleUpdateInput) -> None:
```

Comment:
```suggestion
    def handle_update_validator(self, input: HandleUpdateInput) -> None:
```

I think this shouldn't be async